doc/releases/pacific.rst

```diff
@@ -2,6 +2,239 @@
 Pacific
 =======
 
+v16.2.15 Pacific
+================
+
+This is the fifteenth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* `ceph config dump --format <json|xml>` output will display the localized
+  option names instead of its normalized version. For e.g.,
+  "mgr/prometheus/x/server_port" will be displayed instead of
+  "mgr/prometheus/server_port". This matches the output of the non pretty-print
+  formatted version of the command.
+
+* CEPHFS: MDS evicts clients which are not advancing their request tids which causes
+  a large buildup of session metadata resulting in the MDS going read-only due to
+  the RADOS operation exceeding the size threshold. `mds_session_metadata_threshold`
+  config controls the maximum size that a (encoded) session metadata can grow.
+
+* RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  It's safer replacement is
```

Comment:
only a nit: `due to being`  sounds a bit odd, how about `due to its susceptibility to false negatives.`

Reply:
Agreed.